packages/docs/src/routes/docs/(qwik)/faq/index.mdx

```diff
@@ -2,6 +2,7 @@
 title: FAQ
 contributors:
   - tidiview
+  - hamatoyogi
```

Comment:
we already know you are a contributor to this page Yoav 😂

Reply:
removed dup.